### PR TITLE
Uploaded Successfully

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

**Review by urmilagagare — APPROVED** [review]

**Review by vijaya1811 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by NikitaShivhare1 — APPROVED** [review]

**Review by vijaya1811 — APPROVED** [review]

**Review by jyoti-ing — APPROVED** [review]

File: Ashwin/HTML/01_Assignment/Ashwin_Ass1.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <head>
+        <title>Assignment 1</title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>HTML is a standard markup language for Web Pages</p>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass1.html]: extra line required at end of the file

File: Ashwin/HTML/01_Assignment/Ashwin_Ass3.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<html>
+    <head>
+        <title>Assignment 2</title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+            <h3>Introduction</h3>
+                 <p>HTML is a standard markup language for Web Pages</p>
+        <hr>
+                    <h3>HTML Editor</h3>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass3.html]: check indentation

File: Ashwin/HTML/01_Assignment/Ashwin_Ass3.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+                        <h2>Notepad or TextEdit</h2>
+        <p>Web pages can be created and modified by using professional HTML editors <br>
+            However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(MAC)
+        </p>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass3.html]: extra line required at end of the file

File: Ashwin/HTML/01_Assignment/Ashwin_Ass2.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <head>
+        <title>Assignment 2</title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>HTML is a standard markup language for Web Pages</p>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass2.html]: extra line required at end of the file

File: Ashwin/HTML/01_Assignment/Ashwin_Ass2.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<html>
+    <head>
+        <title>Assignment 2</title>
+    </head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass2.html]: incomplete program please check question again

File: Ashwin/HTML/01_Assignment/Ashwin_Ass4.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        My strength is I am a good Listner. My Hobbies are like playing cricket and cooking.
+        <br>
+        <br>
+        That's all about me.
+        <br>
+        <br>
+        Thank You 
+        </P>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/01_Assignment/Ashwin_Ass4.html]: extra line required at end of the file

File: Ashwin/HTML/02_Assignment/AshwinAss2.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    <head>
+        <title>Assignment 02</title>
+    </head>
+    <body style="background-color: orange">
+        <center>
+            <h1>
+                Google
+            </h1>
+        </center>
+                <pre><em>This article is about the company.</em></pre>
```
Comment (KomalShinde813) [inline Ashwin/HTML/02_Assignment/AshwinAss2.html:9]: check indentation

File: Ashwin/HTML/03_Assignment/Ashwin_contactus.html [hunk 8]
```diff
@@ -0,0 +1,47 @@
+<!-- This page is for contact us page  -->
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_contactus.html]: remove extra line

File: Ashwin/HTML/03_Assignment/Ashwin_contactus.html [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+<head>
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_contactus.html:4]: check iindentation

File: Ashwin/HTML/03_Assignment/Ashwin_contactus.html [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+        }
+        img
+        {
+            height: 600px;
+            width: 400px;
+        }
+    </style>
+</head>
+
+<body style="text-align: center; background:  rgb(71, 99, 22);">
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_contactus.html:37]: check indentation

Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_contactus.html:37]: remove extra space which is present after : in style

File: Ashwin/HTML/03_Assignment/Ashwin_contactus.html [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+    <div style="display: flex; margin-top: 30px;">
+        <div><img src="Images/Background.jpg" alt="Image"></div>
+        <div style="float: left;"><h3>About Lorem :</h3></div>
+        <div style="width: 390px; float: left; margin-top: 20px; margin-left: 30px;"> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Veritatis in quo commodi expedita fugit adipisci
+            repellendus accusamus, quas ex this consectetur magni laborum neque facere eius numquam cumque fugiat
+            ipsa.</div>
+    </div>
+</body>
+
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_contactus.html]: extra line required at end of the file

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 12]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: check indentation

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 13]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: remove extra line

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+            text-decoration: none;
+            text-align: center;
+        }
+
+        a:visited {
+            color: rgb(122, 0, 41);
+        }
+
+        a:hover {
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: remove extra line

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+            color: rgb(175, 255, 71);
+        }
+    </style>
+</head>
+
+<body style="text-align: center; background-color: darkcyan; color: white;">
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: check indentation

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+    <a href="Ashwin_contactus.html">Contact Us</a>
+    <div style="margin-top: 20px; font-size: 30px;">Contact Us</div>
+    <h2 style="float: left;">MI-11-MEAN</h2>
+    
+       <div style="display: inline-block; float:none; margin-top: 50px; margin-left: -140px;">
+        <ul > 
+            <li>Pooja</li>
+            <li>Shubham</li>
+            <li>Ashwin</li>
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: remove extra line

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+        </ul>
+        <ol>
+            <li>shubham</li>
+            <li>darshan</li>
+            <li>shivam</li>
+        </ol></div>
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: remove extra lines

File: Ashwin/HTML/03_Assignment/Ashwin_mean.html [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+        <ol>
+            <li>shubham</li>
+            <li>darshan</li>
+            <li>shivam</li>
+        </ol></div>
+    
+
+</body>
+
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_mean.html]: extra line required at end of the file

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 19]
```diff
@@ -0,0 +1,108 @@
+<!-- Write a html program to create a simple website with 3 pages.Home page,MI-MEAN-9,Contact
+us page. -->
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html:9]: remove extra line

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+        }
+
+        .container {
+            display: flex;
+        }
+
+        img {
+            height: 600px;
+            width: 400px;
+        }
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html]: one line space required after this

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+        .same{
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html]: space required before {

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 22]
```diff
@@ -0,0 +1,108 @@
...
+            margin: 20px;
+            padding: 20px;
+        }
+    </style>
+</head>
+
+<body style="background-color: rgb(71, 99, 22); text-align: center;">
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html:46]: check indentation

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 23]
```diff
@@ -0,0 +1,108 @@
...
+            aliquid error voluptatum quibusdam, ab sequi quas fuga. Esse molestiae eos maxime deserunt ratione
+            perferendis quis animi temporibus magnam laboriosam libero aliquid inventore nobis voluptate natus expedita
+            placeat, eaque cum distinctio reprehenderit ea ipsam, consectetur sunt quam! Mollitia voluptatum voluptates
+            nostrum aspernatur, a deleniti, quam accusantium dicta praesentium, vel expedita assumenda dolore?
+            Reprehenderit beatae ipsum, dolorem quod fugit blanditiis adipisci provident itaque dolore et dolores
+            incidunt veniam similique magni dolor voluptates nobis iste reiciendis eum sit, repellat totam veritatis.
+            .</div>
+            <div class="same"><img src="Images/Background.jpg" alt="assss" style="float: right;"></div>
+        </div>
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html:111]: remove extra lines

File: Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html [hunk 24]
```diff
@@ -0,0 +1,108 @@
...
+            Reprehenderit beatae ipsum, dolorem quod fugit blanditiis adipisci provident itaque dolore et dolores
+            incidunt veniam similique magni dolor voluptates nobis iste reiciendis eum sit, repellat totam veritatis.
+            .</div>
+            <div class="same"><img src="Images/Background.jpg" alt="assss" style="float: right;"></div>
+        </div>
+    
+
+</body>
+
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/03_Assignment/Ashwin_simple_webpage.html:115]: extra line required at end of the file

File: Ashwin/HTML/04_Assignment/Ashwin_01.html [hunk 25]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_01.html]: chnage title, title should be related to program. make this change everywhere

File: Ashwin/HTML/04_Assignment/Ashwin_01.html [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+    <style>
+    .center
+        {
+            margin-left: auto;
+            margin-right: auto;
+        }       
+    </style>
+</head>
+<body>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_01.html]: check indentation

File: Ashwin/HTML/04_Assignment/Ashwin_01.html [hunk 27]
```diff
@@ -0,0 +1,46 @@
...
+    <div>
+    <table class="center" border="1" style="padding: 10px;color: gainsboro; background-color: dimgrey; height: 350px; width: 500px; text-align: center; margin-bottom: auto; margin-right: auto;">
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_01.html]: space required after each style property

File: Ashwin/HTML/04_Assignment/Ashwin_01.html [hunk 28]
```diff
@@ -0,0 +1,46 @@
...
+                <td>Females</td>
+                <td>1.7</td>
+                <td>0.002</td>
+                <td>43%</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_01.html]: extra line required at end of the file

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 29]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: check indentation

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 30]
```diff
@@ -0,0 +1,65 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: change title

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 31]
```diff
@@ -0,0 +1,65 @@
...
+</head>
+<body>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: check indentation

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 32]
```diff
@@ -0,0 +1,65 @@
...
+            </tr>
+            <tr>
+                <td>3:30</td>
+                <td rowspan="3">CSC <br> 205</td>
+                <td rowspan="3">CSC <br> 205</td>
+                <td rowspan="3">CSC <br> 205</td>
+            </tr>
+            <tr>
+                <td>4:00</td>
+                <!-- <td></td> -->
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: remove commented code

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 33]
```diff
@@ -0,0 +1,65 @@
...
+            </tr>
+            <tr>
+                <td>4:30</td>
+                <td></td>
+            </tr>
+        </tbody>
+    </table>
+
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: remove extra line

File: Ashwin/HTML/04_Assignment/Ashwin_02.html [hunk 34]
```diff
@@ -0,0 +1,65 @@
...
+            </tr>
+            <tr>
+                <td>4:30</td>
+                <td></td>
+            </tr>
+        </tbody>
+    </table>
+    
+</body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_02.html]: extra line required at end of the file

File: Ashwin/HTML/04_Assignment/Ashwin_03.html [hunk 35]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_03.html:3]: check indentation

File: Ashwin/HTML/04_Assignment/Ashwin_03.html [hunk 36]
```diff
@@ -0,0 +1,72 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_03.html]: change title

File: Ashwin/HTML/04_Assignment/Ashwin_03.html [hunk 37]
```diff
@@ -0,0 +1,72 @@
...
+        {
+            background-color: violet;
+        }
+        .green
+        {
+            background-color: yellowgreen;
+        }
+    </style>
+</head>
+<body>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_03.html]: check indentation

File: Ashwin/HTML/04_Assignment/Ashwin_03.html [hunk 38]
```diff
@@ -0,0 +1,72 @@
...
+                    <td>Wednesday</td>
+                    <td class="yellow">8.00 am</td>
+                    <td class="green">12.00 pm</td>
+                    <td colspan="3">XSL Formating Object</td>
+                </tr>
+            </tbody>
+        </table>
+    
+</body>
+</html>
```
Comment (KomalShinde813) [inline Ashwin/HTML/04_Assignment/Ashwin_03.html]: extra line required at end of the file

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**